Sources/ApolloCodegenTestSupport/Resources.swift

```diff
@@ -8,11 +8,23 @@ public struct Resources {
 
   public static let GraphQLOperations: [URL] = Bundle.urls(
     forResourcesWithExtension: "graphql",
-    subdirectory: nil
+    subdirectory: "graphql"
   )!
 
   public static let AnimalKingdomSchema = Bundle.url(
     forResource: "AnimalSchema",
-    withExtension: "graphqls"
+    withExtension: "graphqls",
+    subdirectory: "graphql"
+  )!
+
+  public static let CCNGraphQLOperations: [URL] = Bundle.urls(
+    forResourcesWithExtension: "graphql",
+    subdirectory: "ccnGraphql"
+  )!
+
+  public static let CCNAnimalKingdomSchema = Bundle.url(
+    forResource: "AnimalSchema",
+    withExtension: "graphqls",
+    subdirectory: "ccnGraphql"
```

Comment:
I've made a copy of `AnimalKingdomAPI` that has CCN enabled, and then the `.graphql` and the `.graphqls` files from both targets are copied to bundle resources by `ApolloCodegenTestSupport` into separate directories so everything can be tested in `ApolloCodegenTests`.